### DISCUSSION TITLE
Need some inputs regarding full backup failed scenarios

### DISCUSSION THREAD

**Discussion by davidsattenapalli** [olahallengren/sql-server-maintenance-solution#492] [Ideas]
Hi Ola sir,

Thanks a million for writing MaintenanceSolution.SQL and saving thousands of DBAs around the world. Kindly help in the below scenario.

In our organization we have more than 150 databases in one instance and full backup should run for every day. Our backup script should be in such a way that if backup job fails while taking 100th database and if I rerun the job again it should start from the failed database(100th) and exclude the backups which are already taken and the script should take the backup of the remaining databases which are not backuped up for the day. I think the current backup script won't serve this purpose. Please help sir. Thanks in advance.

Regards,
David.S

**Comment by andig1** [comment]
i think it is not possible by Ola's default scripts, hence i would consider the following:
If the Job fail then an additionally step must be executed which check all Databases for missing FULL/DIFF/LOG backup
The remaining Databases must be forwareded to @Databases Parameter from Ola's Script and execute the same.
